provisioning/transport/amqp/src/amqp.ts

```diff
@@ -121,15 +122,27 @@ export class Amqp extends EventEmitter implements X509ProvisioningTransport, Tpm
           /*Codes_SRS_NODE_PROVISIONING_AMQP_16_022: [`disconnect` shall call its callback immediately if the AMQP connection is disconnected.]*/
           disconnect: (callback) => callback()
         },
-        connectingX509: {
+        connectingX509OrSymmetric: {
```

Comment:
>connectingX509OrSymmetric [](start = 8, length = 25)

nit: SymmetricKey. it's long enough we don't care about 3 more letters that make the thing a lot clearer.

Reply:
Done.

---
In reply to: [232119299](https://github.com/Azure/azure-iot-sdk-node/pull/418#discussion_r232119299) [](ancestors = 232119299)